src/auxiliary.jl

```diff
@@ -19,6 +19,23 @@ function set_library!(path = nothing; force = true)
     @info "Please restart Julia and reload OpenFHE.jl for the library changes to take effect"
 end
 
+"""
+    OpenFHE.set_native_int!(native_int::Integer = 64; force = true)
+
+Set the native integer size for an OpenFHE-julia installation, it can be `64` or `128` bits.
+Restart the Julia session after executing this function such that the changes take effect.
+
+See also: [`get_native_int`](@ref)
+"""
+function set_native_int!(native_int::Integer = 64; force = true)
```

Comment:
```suggestion
function set_native_int!(native_int::Integer = 64)
```
Do we ever need `force=false`?